### PR TITLE
Adds Projects, Roles, and Bounties

### PR DESCRIPTION
**Pull request by Flaxscrip**
Adds projects, roles, and bounties as solution to #7

### CODE REVIEW

**Comment by macterra** [comment]
Projects is already here >> https://github.com/cryptotechguru/EDENomicon/tree/master/Projects

**Comment by macterra** [comment]
Bounties is already here >> https://github.com/cryptotechguru/EDENomicon/tree/master/Projects/Bounties

File: README.md [hunk 1]
```diff
@@ -25,3 +25,16 @@
     1. Issues SHOULD be reviewed by the Operators to determine whether the rule should be changed or the player should be sanctioned.
 1. Only players may fill [roles](Roles/).
 1. If a proposal offers a choice between two or more options an Operator should call a [multi-vote](multi-vote.md).
+
+## [Projects](Projects/)
```
Comment (macterra) [inline README.md:29]: This reverts the change I made in https://github.com/cryptotechguru/EDENomicon/commit/747b6f8863177a66339ce473f27b095b89e945e5

**Comment by genecyber** [comment]
looks good

**Pull request merged**